stl/inc/atomic

```diff
@@ -1666,11 +1724,13 @@ template <class _Ty>
 _CXX20_DEPRECATE_ATOMIC_INIT void atomic_init(
     volatile atomic<_Ty>* const _Mem, const typename atomic<_Ty>::value_type _Value) noexcept {
     // NB: respecting volatility here appears unimplementable
+    static_assert(_Deprecate_non_lock_free_volatile<_Is_always_lock_free<sizeof(_Ty)>>, "Never fails");
     _STD atomic_init(const_cast<atomic<_Ty>*>(_Mem), _Value);
 }
 
 template <class _Ty>
 void atomic_store(volatile atomic<_Ty>* const _Mem, const _Identity_t<_Ty> _Value) noexcept {
+    static_assert(_Deprecate_non_lock_free_volatile<_Is_always_lock_free<sizeof(_Ty)>>, "Never fails");

```

Comment:
Note to other reviewers: Despite that WG21-P1831 doesn't mention these other non-member functions explicitly, they are specified as equivalent to calling the (now conditionally deprecated) member functions in [atomics.nonmembers]/1, so they should be deprecated.